examples/with-postgres/README.md

```diff
@@ -0,0 +1,73 @@
+# Postgres.js Example
+
+An example using [Postgres.js](https://github.com/porsager/postgres) in a Next.js project.
+
+## Deploy your own
+
+Once you have access to [the environment variables you'll need](#configure-environment-variables), deploy the example using [Vercel](https://vercel.com?utm_source=github&utm_medium=readme&utm_campaign=next-example):
+
+[![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/git/external?repository-url=https://github.com/vercel/next.js/tree/canary/examples/with-postgres&project-name=with-postgres&repository-name=with-postgres&env=DATABASE_URL&envDescription=Required%20to%20connect%20the%20app%20with%20Postgres)
+
+## How to use
+
+Execute [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app) with [npm](https://docs.npmjs.com/cli/init), [Yarn](https://yarnpkg.com/lang/en/docs/cli/create/), or [pnpm](https://pnpm.io) to bootstrap the example:
+
+```bash
+npx create-next-app --example with-postgres with-postgres-app
+```
+
+```bash
+yarn create next-app --example with-postgres with-postgres-app
+```
+
+```bash
+pnpm create next-app --example with-postgres with-postgres-app
+```
+
+## Configuration
+
+### Install dependencies
```

Comment:
This should already be done by the create next-app CLI, so probably safe to remove

Reply:
Makes sense, removing...